src/Storage/Clickhouse.php

```diff
@@ -51,7 +51,7 @@ public function last($count, DiagRecord $beforeRecord = null): array
         ";
 
         if (null !== $beforeRecord) {
-            $sql .= " where createdAt <  '" . $beforeRecord->getCreatedAt()->format('Y-m-d H:i:s') . "'";
```

Comment:
no need to cast for (string) in case of concatenation with string  -> https://3v4l.org/MqKZI